paparazzi-gradle-plugin/src/test/projects/multiplatform-plugin-with-android/gradle.properties

```diff
@@ -0,0 +1 @@
+kotlin.mpp.androidSourceSetLayoutVersion1.nowarn=true
```

Comment:
is this one we should fix instead?

Reply:
If you know how :), there's no code in that project just some plugin applied.

Reply:
i don't.  alright then, figured i'd ask :)